dot/utils.go

```diff
@@ -143,20 +148,19 @@ func NewTestConfig(t *testing.T) *Config {
 	return cfg
 }
 
-// newTestConfigWithFile returns a new test configuration and a temporary configuration file
-func newTestConfigWithFile(t *testing.T) (*Config, *os.File) {
+// NewTestConfigWithFile returns a new test configuration and a temporary configuration file
+func NewTestConfigWithFile(t *testing.T) (*Config, *os.File) {
```

Comment:
Why is this exported?

Reply:
No reason, changed back to `newTestConfigWithFile`